action-scripts/brownie/networks.yaml

```diff
@@ -22,3 +22,27 @@ live:
         host: https://rpcapi.fantom.network
         id: fantom-main
         name: Mainnet
+
```

Comment:
I think if you pull our newest fork branch of brownie these should be here.  Would be good to keep those up to date too. I am not sure anything uses this file, but maybe you set that up at some point.

Reply:
they are not there, but i can pr them later